### DISCUSSION TITLE
how to get my answer accepted ?

### DISCUSSION THREAD

**Discussion by raeef1001** [ParanoidUser/yolo#916] [Ask Away]
how to get my answer accepted  ?

**Comment by ParanoidUser** [comment]
Well, the rules are simple and straightforward:
1. Questions are posted daily.
2. The best answer (in terms of accuracy, completeness, and uniqueness) is picked the next day.
3. If the answers are too similar, the person who answered the question first wins (in some cases, an additional question may be asked).
